q2_deblur/_denoise.py

```diff
@@ -147,6 +147,14 @@ def _denoise_helper(
                                              pd.DataFrame):
     _check_inputs(**locals())
     with tempfile.TemporaryDirectory() as tmp:
+        df = demultiplexed_seqs.manifest.view(pd.DataFrame)
```

Comment:
Please move this guard up to just below `_check_inputs` --- this will keep similar code grouped together.